src/app/core/services/common/accounting-export.service.ts

```
@@ -72,6 +72,10 @@ export class AccountingExportService {
     }
 
     if (appName === AppName.QBD_DIRECT) {
+      if (apiParams.status__in?.includes(AccountingExportStatus.FAILED)) {
+        apiParams.status__in = AccountingExportStatus.ERROR;
+      }
+      delete apiParams.type__in;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)